### DISCUSSION TITLE
Editing original post moves 'creation time' forward

### DISCUSSION THREAD

**Discussion by Griatch** [community/community#2957] [Discussions] [answered]
When editing the start-post of a thread, the text `Griatch started 10h ago` is updated to reflect the time of my edit (so it becomes `Griatch started 1m ago`). 
I don't think this is intuitive - the start-time should remain the point at which the thread was originally created. Rather, #2951 would (in my view) be the way to indicate that the thread has changed (but there is probably no point in having that include edits either).

**Comment by becca** [comment, marked as answer]
This was a bug reported over in https://github.com/github/feedback/discussions/2855. We pushed a fix for this bug this morning 🚀
